docs/pre-release/install/helm.md

```diff
@@ -51,13 +51,107 @@ clusters:
+
+For example, suppose you want a Traffic Manager that only works on namespaces `dev` and `staging`.
+To do this, create a `values.yaml` like the following:
+
+```yaml
+managerRbac:
+  create: true
+  namespaced: true
+  namespaces:
+  - dev
+  - staging
+```
+
+This can then be installed via:
+
+```bash
+helm install traffic-manager --namespace staging datawire/telepresence -f ./values.yaml
+```
+
+**NOTE** Do not install namespace-scoped Traffic Managers and a global Traffic Manager in the same cluster, as it could have unexpected effects.
+
+#### Namespace collision detection
+
+The Telepresence Helm chart will try to prevent namespace-scoped Traffic Managers from managing the same namespaces.
+It will do this by creating a ConfigMap, called `traffic-manager-claim`, in each namespace that a given install manages.
+
+So, for example, suppose you install one Traffic Manager to manage namespaces `dev` and `staging`, as:
+
+```bash
+helm install traffic-manager --namespace dev datawire/telepresence --set 'managerRbac.namespaced=true' --set 'managerRbac.namespaces={dev,staging}'
+```
+
+You might then attempt to install another Traffic Manager to manage namespaces `staging` and `prod`:
+
+```bash
+helm install traffic-manager --namespace prod datawire/telepresence --set 'managerRbac.namespaced=true' --set 'managerRbac.namespaces={staging,prod}'
+```
+
+This would fail with an error:
+
+```
+Error: rendered manifests contain a resource that already exists. Unable to continue with install: ConfigMap "traffic-manager-claim" in namespace "staging" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "prod": current value is "dev"
+```
+
+To fix this error, fix the overlap either by removing `staging` from the first install, or from the second.
+
+#### Namespace scoped user permissions
+
+Optionally, you can also configure user rbac to be scoped to the same namespaces as the manager itself.
+You might want to do this if you don't give your users permissions throughout the cluster, and want to make sure they only have the minimum set required to perform telepresence commands on certain namespaces.
+
+Continuing with the `dev` and `staging` example from the previous section, simply add the following to `values.yaml` (make you set the `subjects`!):
+
+```yaml
+clientRbac:
+  create: true
+
+  # These are the users or groups to which the user rbac will be bound.
+  # This MUST be set.
+  subjects: {}
+  # - kind: User
+  #   name: jane
+  #   apiGroup: rbac.authorization.k8s.io
+
+  namespaced: true
+
+  namespaces:
+  - dev
+  - staging
+```
+
+#### Namespace-scoped webhook
+
+If you wish to use the traffic-manager's [mutating webhook](../reference/cluster-config#mutating-webhook) with a namespace-scoped traffic manager, you will have to ensure that each namespace has an `app.kubernetes.io/name` label that is identical to its name:
```

Comment:
I think might also be worth saying something like: You can also use `kubectl label ...` to add the label to a pre-existing namespace